packages/hub/src/session.ts

```diff
@@ -115,7 +115,7 @@ export class Session implements SessionInterface {
       did: typeof this.did === 'number' || typeof this.did === 'string' ? `${this.did}` : undefined,
       duration: this.duration,
       attrs: dropUndefinedKeys({
-        release: this.release,
+        release: typeof this.release !== 'string' ? JSON.stringify(this.release) : this.release,
```

Comment:
Most likely this fix should not be limited to sessions.
Can we make it so that we handle any event going out with a release? (errors, transactions, sessions)

Reply:
That's what I thought as well but @kamilogorek  told me that we should probably not change any of the others until they come up because he is never seen this release object issue happen anywhere else